css/style.scss

```diff
@@ -0,0 +1,244 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
+	background-color: #222222;
+}
+
+html,
+body {
+	width: 100%;
+	height: 100%;
+	margin: 0;
+	padding: 0;
+}
+
+body {
+	background: transparent;
+	color: #fff;
+	font-size: 18px;
+	line-height: 28px;
+	font-weight: 300;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", sans-serif;
+}
+
+.container {
+	width: 100%;
+	max-width: 1280px;
+	margin: 0 auto;
+
+	padding: 0 20px;
+
+	display: block;
+}
+
+.content {
+	width: 100%;
+	margin: 100px auto 0;
+
+	display: flex;
+	align-items: center;
+	flex-wrap: nowrap;
+
+	position: relative;
+
+	@media screen and (max-width: 1024px) {
+		margin: 40px auto 0;
+
+		flex-wrap: wrap;
+	}
+
+	&__left {
+		width: 55%;
+		padding-right: 20px;
+
+		@media screen and (max-width: 1024px) {
+			width: 100%;
+			max-width: 480px;
+
+			padding-right: 0;
+			padding-bottom: 30px;
+		}
+	}
+
+	&__right {
+		width: 45%;
+
+		position: relative;
+
+		@media screen and (max-width: 1024px) {
+			width: 100%;
+			margin-left: 10px;
+			padding-bottom: 40px;
+		}
+
+		&::before {
+			background-color: #ff0000;
+
+			width: 128px;
+			height: 20px;
+
+			margin: 0;
+			display: block;
+
+			position: absolute;
+			top: -10px;
+			left: -10px;
+
+			content: '';
+
+			z-index: -10;
+
+			@media screen and (max-width: 1024px) {
+				width: 92px;
+			}
+		}
+	}
+}
+
+.socials {
+	width: 100%;
+	margin: 0 auto 20px;
+
+	padding-top: 40px;
+
+	display: block;
+	position: relative;
+
+	&::before {
+		background-color: #ffffff;
+
+		width: 50%;
+		max-width: 256px;
+
+		height: 4px;
+		margin: 0;
+		display: block;
+
+		position: absolute;
+		top: 0;
+		left: 0;
+
+		content: '';
+	}
+
+	ul,
+	li {
+		width: 100%;
+		margin: 0;
+		padding: 0;
+		display: block;
+		list-style: none;
+	}
+
+	ul {
+		margin-bottom: 20px;
+	}
+
+	li {
+		margin-bottom: 20px;
+
+		&:last-of-type {
+			margin-bottom: 0;
+		}
+
+		&.location,
+		&.contact {
+			background-repeat: no-repeat;
+			background-position: 2px 50%;
+			background-size: 14px;
+
+			padding-left: 30px;
+
+			font-size: 20px;
+			line-height: 30px;
+			font-weight: 300;
+		}
+
+		&.location {
+			background-image: url('../img/icon-location.svg');
+		}
+
+		&.contact {
+			background-image: url('../img/icon-email.svg');
+		}
+	}
+}
+
+img {
+	width: 100%;
+	max-width: 256px;
+	margin: 0;
+	display: block;
+}
+
+h1 {
+	margin: 0 auto 20px;
+
+	color: #ff0000;
+	font-size: 60px;
+	font-weight: 700;
+	line-height: 60px;
+
+	@media screen and (max-width: 1024px) {
+		font-size: 40px;
+		line-height: 40px;
+	}
+}
+
+h2 {
+	margin: 0 auto 20px;
+
+	color: #ffffff;
+	font-size: 60px;
+	font-weight: 300;
+	line-height: 60px;
+
+	@media screen and (max-width: 1024px) {
+		font-size: 40px;
+		line-height: 40px;
+	}
+}
+
+p {
+	margin: 0 auto 20px;
+
+	color: #ffffff;
+	font-size: 18px;
+	line-height: 28px;
+}
+
+.button {
```

Comment:
Add smooth transition to button hover.
It's kind of weird right now with just a border.